packages/plugins/ansiblerole-insights-client/ansiblerole-insights-client.spec

```diff
@@ -0,0 +1,56 @@
+
+%global repo_orgname RedHatInsights
+%global repo_name insights-client-role
+%global repo_version 1.5
+
+%global role_orgname %{repo_orgname}
+%global role_name insights-client
+%global role_version %{repo_version}
```

Comment:
The repo_xxx vars are git repo names, the role_xxx vars are the names you want installed on disk.

In this case the repo_name (insights-client-role) must be different from the role_name (insights-client).  The repo_name must be insights-client-role because the git repo insight-client contains the actual code that this role installs.  But we don't want the role's name to be insights-client-role, that would be redundant and silly.  Galaxy allows you to change the role's name as you import it, and we are just echoing that functionality here.

As for {repo,role}_orgname.  For this spec file i followed the design of the rhel-system-roles.spec which had this same abiltiy to have a different orgname/namespace from the repo to the role.  In my case i felt the RedHatInsights org name of the repo worked find as an Namespace for the role, but i left the possiblity of changing it in case we needed to.

As for the {repo,role}_version, i don't see any reason to change the version from repo to role, and have to admit i just continued the pattern without thinking about it.

Reply:
Sorry, i read this wrong, `role` v `repo` :man_facepalming: